dev/com.ibm.ws.st.core_tests/bvt/src/com/ibm/ws/st/core/tests/jee/CDIRequiredFeaturesBeans20.java

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.st.core.tests.jee;
+
+import org.eclipse.core.runtime.Path;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.AllTests;
+
+import com.ibm.ws.st.core.internal.WebSphereRuntime;
+import com.ibm.ws.st.core.internal.WebSphereServer;
+import com.ibm.ws.st.core.tests.util.FeatureUtil;
+import com.ibm.ws.st.tests.common.util.TestCaseDescriptor;
+
+import junit.framework.TestSuite;
+
+/**
+ * Test that correct cdi feature enabled for beans.xml that specifies the 1.1
+ * schema location.
+ */
+@TestCaseDescriptor(description = "CDI required feature for beans.xml with 1.1 schema location", isStable = true)
+@RunWith(AllTests.class)
+public class CDIRequiredFeaturesBeans20 extends CDIRequiredFeaturesBase {
+    private static final String TEST_NAME = "CDIRequiredFeaturesBeans20";
```

Comment:
Should we use the class name so that test output would be updated if the class is renamed in the future? I realize the TEST_NAME is also being used for project import but if someone renames the class the test would fail and they'd realize the import project may need to be updated as well.

Reply:
Sure.  I updated all of the CDI tests.